### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by albemlee** [orionthunter/date-a-scientist#1]
### Criteria 1: Valid Python Code
* _Score Level:_ 4/4
* _Comment(s):_ Code included runs without any errors.

Note: Line 242 (commented out) has a missing parenthesis.
 
### Criteria 2: Exploration of Data
* _Score Level:_ 3/4
* _Comment(s):_ Data is explored somewhat, and the experimental question(s) chosen are logical and based on the data exploration. Features chosen to answer the question make sense, but some of the most obvious features are left out/not addressed.

You did not explore any of the free-form text data. Word clouds are a great way to visualize text. Also think about how you can turn free-form text into features (vectorization, converting to categorical through clustering, etc)
 
### Criteria 3: Machine Learning Techniques used correctly
* _Score Level:_ 4/4
* _Comment(s):_ Algorithms are used correctly and the correct conclusions are drawn from the results.

Great job experimenting with multiple algorithms, and tuning hyperparameters for SVM. I encourage you to look in [XGBoost](https://xgboost.readthedocs.io/en/latest/python/python_api.html) or [Random Forest](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html), which are very popular supervised learning methods right now.

### Criteria 4: Report: Are conclusions clear and supported by data?
* _Score Level:_ 4/4
* _Comment(s):_ Question(s) are stated clearly. The results of 2 regression algorithms and 2 classification algorithms are shown. Conclusions are clearly stated and based on evidence.

Again, great job. One question worth considering: Is accuracy the best way to evaluate classification methods (why not recall, precision, or f-1 score)?

### Criteria 5: Code formatting
* _Score Level:_ 4/4
* _Comment(s):_ Code is formatted clearly and readable.

Consider putting your code and conclusions in a Jupyter notebook and uploading it onto Github. This is a great way to showcase your work to employers.
 
### Overall Score: 19/20
Overall, great job on this project. On any machine learning project, there are many ways to continue to develop and improve your project. I would encourage you to play more with the data and experiment with algorithms that MLF did not cover. 

Most importantly, you demonstrated an understanding of the overall process of machine learning. The framework you used can be applied to any ML project regardless of complexity, and I'm excited to see what you do with it.
